documentation/assembly-pdf.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+  <id>pdf</id>
+  <formats>
+    <format>zip</format>
+  </formats>
+  <includeBaseDirectory>true</includeBaseDirectory>
+  <fileSets>
+    <fileSet>
+      <directory>target/generated-docs-pdf</directory>
+      <includes>
+        <include>*.pdf</include>
+      </includes>
+      <excludes>
+        <exclude>0-glossary.pdf</exclude>
```

Comment:
Why are these files being generated if we don't need them?

Reply:
That was left-over from copy-paste, sorry. Removed